data/nlu/nlu.md

```diff
@@ -4846,6 +4859,11 @@
 - Core
 - CORE
 
+## synonym:custom actions
+- actions
+- Custom actions
+- Actions
```

Comment:
Agreed this is a good idea bc it'll improve greg's search. did you add this?

Reply:
Yes, I indeed added this with Greg's search update in mind. I did not yet test with Greg's search, but plan to do so once that one is merged.